src/containers/UsersList.js

```diff
@@ -83,12 +93,23 @@ export class UsersList extends Component {
             <Grid.Row>
               <Grid.Column width={12}>
                 <Header as='h1'>Volunteers Directory</Header>
+                <Input
+                  onChange={this.handleSearchInput}
+                  className='users-list-search-input'
+                  fluid
+                  icon='search'
+                  placeholder='Search...'
+                />
                 <Card.Group centered itemsPerRow={3}>
-                  <Paginate
-                    items={usersList}
-                    Component={User}
-                    pageCount={pageCount}
-                  />
+                  {usersList ? (
```

Comment:
I might be missing something, is the `userList` check enough to check if there are more than one page of user objects?

Reply:
at the moment @arku, @bryszard we are using this in the `ProjectsList` to not display the `PaginationLinks` if their is only one page, I believe https://github.com/AgileVentures/WebsiteOne-FE/blob/develop/src/containers/ProjectsList.js#L132

maybe you could try that out @bryszard when you get a chance?

Reply:
Thanks for that, I will add it.